dtfs-central-api/src/v1/validation/utilisation-report-service/utilisation-report-validator.js

```diff
@@ -0,0 +1,94 @@
+const { UTILISATION_REPORT_HEADERS } = require('../../../constants/utilisationReportHeaders');
+const {
+  validateUkefId,
+  validateExporter,
+  validateBaseCurrency,
+  validateFacilityUtilisation,
+  validateTotalFeesAccrued,
+  validateMonthlyFeesPaid,
+  validatePaymentCurrency,
+  validateExchangeRate,
+} = require('./utilisation-data-validator');
+
+/**
+ * Validates the month of the utilisation report. Returns null if valid, otherwise returns an error message.
+ * @param {Integer} month - Month of utilisation report.
+ * @returns {String | null} - Error message or null if valid.
+ */
+const validateMonth = (month) => {
+  if (!month) {
+    return 'Month is required';
+  }
+  if (month < 1 || month > 12) {
+    return 'Month must be between 1 and 12';
+  }
+  return null;
+};
+
+/**
+ * Validates the year of the utilisation report. Returns null if valid, otherwise returns an error message.
+ * @param {Integer} year - year of utilisation report.
+ * @returns {String | null} - Error message or null if valid.
+ */
+const validateYear = (year) => {
+  if (!year) {
+    return 'Year is required';
+  }
+  if (year < 2020 || year > 2100) {
+    return 'Year must be between 2020 and 2100';
+  }
+  return null;
+};
+
+/**
+ * Validates the file path of the utilisation report in azure. Returns null if valid, otherwise returns an error message.
+ * @param {Integer} filePath - file path of the utilisation report in azure.
+ * @returns {String | null} - Error message or null if valid.
+ */
+const validateFilePath = (filePath) => {
+  if (!filePath) {
+    return 'File path is required';
+  }
+  if (filePath) {
+    if (typeof filePath !== 'string') {
+      return 'File path must be a string';
+    }
+  }
+  return null;
+};
+
+/**
+ * Validates the utilisation report data. Returns an array of error messages.
+ * @param {Array} utilisationReportData - array of json objects representing utilisation report data.
+ * @returns {Array} - Array of string error messages.
```

Comment:
We could be more specific about the types here, using `Array<object>` and `Array<string>`. It looks like the `object[]` and `string[]` is more commonly used in the project though.